app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str):
+    cmd_list = command.split()
+    cmd = cmd_list[0]
+    old_file = cmd_list[1]
+    new_file = cmd_list[2]
+
+    if old_file == new_file and cmd != "cp":
```

Comment:
You'd also want to use `or` operator here, not `and`.

Reply:
thx)